bat.go

```diff
@@ -60,6 +62,8 @@ func init() {
 	flag.BoolVar(&form, "f", false, "Submitting as a form")
 	flag.BoolVar(&download, "download", false, "Download the url content as file")
 	flag.BoolVar(&download, "d", false, "Download the url content as file")
+	flag.BoolVar(&insecureSSL, "insecure", false, "Allow connections to SSL sites without certs")
+	flag.BoolVar(&insecureSSL, "k", false, "Allow connections to SSL sites without certs")
```

Comment:
if this set to k it will inconsistent witth other. like `download` short name is `d`. So I think `i` will make sense